README.md

```diff
@@ -0,0 +1,50 @@
+# Supplementary Policies for Apache Cassandra Java Driver
+Currently, this package contains two load balancing policies to be used with the [Apache Cassandra Java Driver](https://github.com/apache/cassandra-java-driver) (version 4.10.0+) - `LatencyAndInflightCountLoadBalancingPolicy` and `LatencySensitiveLoadBalancingPolicy`.
+
+## Get Started
+To use either of the policies, you need to include both the Java driver and this package as dependencies.
+```xml
+        <dependency>
+            <groupId>org.apache.cassandra</groupId>
+            <artifactId>java-driver-core</artifactId>
+            <version>4.10.0</version>
+        </dependency>
```

Comment:
The Maven coordinates for this dependency are wrong.  4.10.0 was a DataStax release so it's groupId would still be com.datastax.oss.